packages/node_modules/@webex/plugin-meetings/src/members/index.js

```diff
@@ -668,4 +689,39 @@ export default class Members extends StatelessWebexPlugin {
 
     return this.membersRequest.transferHostToMember(options);
   }
+
+
+  /**
+   * Sends DTMF tones for the PSTN member of a meeting
+   * @param {String} tones a string of one or more DTMF tones to send
+   * @param {String} memberId member id
+   * @returns {Promise}
+   * @public
+   * @memberof Members
+   */
+  sendDTMF(tones = '', memberId = '') {
```

Comment:
DTMF might not be a term that many people outside telephony would know
```suggestion
  sendDialPadKey(tones = '', memberId = '') {
```